### DISCUSSION TITLE
Modeling visibility due to fog formation   ( Condensation of water vapor in air )

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13825] [Q&A]
Is it possible to check the impairment of visibility due to FOG formation ( condensation of water vapor in air ) as a result of  cold CO2 jet release with dry ice sublimation in air ? With the existing visibility model inside FDS , is it possible to  tweak some constants to get a ball park range of visibility due to fog formation ?

**Comment by drjfloyd** [comment]
Please read the User's Guide. It discusses the various options available to you for VISIBILITY.

**Comment by bwklein** [comment]
@drjfloyd To clarify the question...
The SPEC that we want to use for the Visibility calculation is 'WATER VAPOR_COND' that is generated by FDS when AEROSOL=T on the 'WATER VAPOR' SPEC line.  I don't see a way to pass the MASS_EXTINCTION_COEFFICIENT parameter into this FDS generated species that we could then refer to with SPEC_ID reference for the 'VISIBILITY' quantity, or as the Species to be used for SM3D output.

The hope is to use the 'WATER VAPOR_COND' species for visibility calculations and in 3D Results as a 'fog' or vapor cloud.

**Reply by drjfloyd** [reply]
Add it to the WATER VAPOR input.  WATER_VAPOR_COND should inherit everything from WATER VAPOR. Since WATER VAPOR isn't being used for the visibility calc, the fact that it has an extinction coefficient should just be ignored.

**Reply by bwklein** [reply]
One other problem is that during initialization any reference to 'WATER VAPOR_COND' throws an error because the SPEC is not defined in the input file.

**Reply by drjfloyd** [reply]
@bwklein can you open an issue with a simple input. It is a tracked species created when SPEC are processed so it should be seen by the various outputs.

**Reply by bwklein** [reply]
@drjfloyd I think I have resolved supporting it in PyroSim and FDS. I believe that it is working as you described.  The problem has moved to viewing the SM3D data in Smokeview and in PyroSim Results Viewer.  It doesn't show up at all in our Results viewer and then seems to get clipped to transparent in Smokeview.  I'll start an issue with a file for @gforney to look at when he gets a chance.

@Muthu-V ^

**Comment by bwklein** [comment]
Issue created: https://github.com/firemodels/fds/issues/13840
